Library/StripeTypes.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import KsApi
+
+public protocol StripeSCARequiring {
+  var requiresSCAFlow: Bool { get }
+  var clientSecret: String? { get }
+}
+
+extension UpdateBackingEnvelope: StripeSCARequiring {
+  public var requiresSCAFlow: Bool {
+    return self.updateBacking.checkout.backing.requiresAction
+  }
+
+  public var clientSecret: String? {
+    return self.updateBacking.checkout.backing.clientSecret
+  }
+}
+
+extension CreateBackingEnvelope: StripeSCARequiring {
+  public var requiresSCAFlow: Bool {
+    return self.createBacking.checkout.backing.requiresAction
+  }
+
+  public var clientSecret: String? {
+    return self.createBacking.checkout.backing.clientSecret
+  }
+}
+
+public enum StripePaymentHandlerActionStatus {
```

Comment:
This type maps to `STPPaymentHandler.ActionStatus` and makes it easier for us to mock in tests without having to link the Stripe SDK to our Library framework.